stack_master.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.1.0"
   spec.platform      = gem_platform
 
-  spec.add_development_dependency "bundler", "~> 1.5"
+  spec.add_development_dependency "bundler"
```

Comment:
Should we just change this to `> 1.5` to avoid having to support earlier versions?

Reply:
It's just a development dependency, this change won't influence the released gem.

I think there's only a minute chance that there're Stack Master developers with such a old version of Bundler installed.